seqr/views/pages/staff/discovery_sheet.py

```diff
@@ -503,3 +503,10 @@ def generate_rows(project, errors):
             rows.append(row)
 
     return rows
+
+
+def _load_dict(s):
+    try:
+        return json.loads(s)
+    except ValueError:
```

Comment:
what is "it" in this context?

Reply:
If there are Individual records where i.phenotips_data causes json.loads(s) to throw a ValueError, we should fix/update those i.phenotips_data values and just use json.loads directly instead of _load_dict(..)